os/include/tinyara/pm/pm.h

```diff
@@ -314,9 +308,35 @@ enum pm_timer_type_e {
 };
 
 struct pm_timer_s {
```

Comment:
It appears to be unused. Please remove any unnecessary definitions.

Reply:
it is used to set lock or wakeup timer